src/snowcli/cli/registry/commands.py

```diff
@@ -14,10 +18,115 @@
 
 
 @app.command("token")
-@with_output
 @global_options_with_connection
-def token(environment: str = ConnectionOption, **options):
+def token(**options):
     """
     Gets the token from environment to use for authenticating with the registry.
     """
-    return ObjectResult(get_token(environment))
+    sys.stdout.write(json.dumps(RegistryManager().get_token()))
```

Comment:
```suggestion
    return MessageResult(json.dumps(RegistryManager().get_token()))
```

Reply:
Or if you want it to be a json then stick to previous implementation but use the command with `--fromat=json`

Reply:
Fixed this one.